pandas/core/strings.py

```diff
@@ -646,9 +646,13 @@ def str_split(arr, pat=None, n=None, return_type='series'):
     """
     from pandas.core.series import Series
     from pandas.core.frame import DataFrame
+    from pandas.core.index import Index
 
     if return_type not in ('series', 'frame'):
```

Comment:
I think we should accept ``'index'`` here (as an alias for series), pls update the doc-string as well.